### PR TITLE
Regenerate object-appearance doc + thumbnails

### PR DESCRIPTION
**Pull request by DavidMansolino**
**Description**
ditto

### CODE REVIEW

**Comment by DavidMansolino** [comment]
The test are Failing because I the new blog post, the thumbnail where set manually without including the original files, I will fix this.

File: docs/guide/object-floors.md [hunk 1]
```diff
@@ -157,6 +157,8 @@ UnevenTerrain {
   SFBool     flatCenter     FALSE
   SFBool     flatBounds     FALSE
   SFInt32    perlinNOctaves 3
+  deprecatedField MFString   texture        "textures/sand.jpg"
+  deprecatedField SFVec2f    textureScale   1.0 1.0
```
Comment (omichel) [inline docs/guide/object-floors.md:161]: I am not sure we should document the deprecated fields...
```suggestion
```

Comment (DavidMansolino) [reply docs/guide/object-floors.md:161]: Good point, I will improve the script to avoid this.

Comment (DavidMansolino) [reply docs/guide/object-floors.md]: This is now done in cc4cfddbd34415909867a40d3222fae702fdb888.

**Pull request merged**